integrations/firebase_crashlytics_demo_project/README.md

```diff
@@ -0,0 +1,16 @@
+# firebase_crashlytics_demo_project
+
+A new Flutter project.
+
+## Getting Started
+
+This project is a starting point for a Flutter application.
+
+A few resources to get you started if this is your first Flutter project:
+
+- [Lab: Write your first Flutter app](https://flutter.dev/docs/get-started/codelab)
+- [Cookbook: Useful Flutter samples](https://flutter.dev/docs/cookbook)
+
+For help getting started with Flutter, view our
+[online documentation](https://flutter.dev/docs), which offers tutorials,
+samples, guidance on mobile development, and a full API reference.
```

Comment:
This looks like the generic Flutter README text. Use an example from one of the other demo projects to tell people what this project is, and to look at the codemagic.yaml

Reply:
Done!